xbmc/addons/binary-addons/BinaryAddonManager.cpp

```diff
@@ -55,6 +57,20 @@ bool CBinaryAddonManager::Init()
   for (auto addon : binaryAddonList)
     AddAddonBaseEntry(addon);
 
+  CServiceBroker::GetAddonMgr().RegisterAddonMgrCallback(ADDON_ADSPDLL, this);
+  CServiceBroker::GetAddonMgr().RegisterAddonMgrCallback(ADDON_AUDIODECODER, this);
+  CServiceBroker::GetAddonMgr().RegisterAddonMgrCallback(ADDON_AUDIOENCODER, this);
+  //CServiceBroker::GetAddonMgr().RegisterAddonMgrCallback(ADDON_GAMEDLL, this);
+  CServiceBroker::GetAddonMgr().RegisterAddonMgrCallback(ADDON_IMAGEDECODER, this);
+  CServiceBroker::GetAddonMgr().RegisterAddonMgrCallback(ADDON_INPUTSTREAM, this);
+  CServiceBroker::GetAddonMgr().RegisterAddonMgrCallback(ADDON_PERIPHERALDLL, this);
+  //CServiceBroker::GetAddonMgr().RegisterAddonMgrCallback(ADDON_PVRDLL, this);
+  CServiceBroker::GetAddonMgr().RegisterAddonMgrCallback(ADDON_SCREENSAVER, this);
+  CServiceBroker::GetAddonMgr().RegisterAddonMgrCallback(ADDON_VFS, this);
+  CServiceBroker::GetAddonMgr().RegisterAddonMgrCallback(ADDON_VIDEOCODEC, this);
+  CServiceBroker::GetAddonMgr().RegisterAddonMgrCallback(ADDON_VIZ, this);
```

Comment:
Register added here, the ADDON_GAMEDLL and ADDON_PVRDLL are commented out until this interfaces are changed.